cypress/support/pages/article.pageObject.js

```diff
@@ -0,0 +1,68 @@
+import PageObject from '../PageObject';
+
+class ArticlePageObject extends PageObject {
+  url = '#/editor';
+
+  elements = {
+    titleField: () => cy.getByDataCy('article-title-field'),
+    descriptionField: () => cy.getByDataCy('article-description-field'),
+    bodyField: () => cy.getByDataCy('article-body-field'),
+    tagsField: () => cy.get('input[data-cy="article-tags-field"]'),
+    publishArticleButton: () => cy.getByDataCy('publish-article-button'),
+    editButton: () => cy.getByDataCy('edit-article-button'),
+    deleteButton: () => cy.getByDataCy('delete-article-button'),
+    usernameLink: () => cy.getByDataCy('username-link')
+  };
+
+  typeTitle(title) {
+    this.elements.titleField().type(title);
+  }
+
+  typeDespription(description) {
```

Comment:
There is a typo in the method name `typeDespription`. It should likely be `typeDescription` to accurately reflect the action being performed.